freeipa.spec.in

```diff
@@ -320,7 +320,8 @@ Requires: selinux-policy >= %{selinux_policy_version}
 Requires(post): selinux-policy-base >= %{selinux_policy_version}
 Requires: slapi-nis >= %{slapi_nis_version}
 Requires: pki-ca >= 10.4.0-1
-Requires: pki-kra >= 10.4.0-1
+# pki-kra 10.5.x fixes vaults in python3
```

Comment:
I don't think we need to include this comment.  The reason is given in the commit message,
and will be superseded by some other reason in the next version bump.

Reply:
The comments are usually added for the RHEL package maintainer so that they know why a certain bump was added there without having to go through git history. But it may not be necessary here, @pvomacka, you're the RHEL magician, what do you think?

Reply:
From my point of view, it makes sense to put there a comment, why package version bump was done. It may save time during RHEL packaging.

Reply:
It doesn't make sense to have different version from pki-ca and pki-kra, though. You won't be able to install different versions. Let's keep the versions in sync.